src/js/components/devices/dialogs/make-gateway-dialog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+// material ui
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from '@mui/material';
+
+import { getToken } from '../../../auth';
+import CopyCode from '../../common/copy-code';
+
+export const getCode = () => `JWT_TOKEN='${getToken()}'
+
+wget -O- https://get.mender.io | sudo bash -s -- --jwt-token $JWT_TOKEN mender-gateway --hosted-mender-gateway --demo`;
```

Comment:
Sorry that I forgot to update the reference doc, the `--hosted-mender-gateway` flag was not used after all. This command should be:

```suggestion
wget -O- https://get.mender.io | sudo bash -s -- --jwt-token $JWT_TOKEN mender-gateway --demo`;
```

Reply:
Thanks! should now be removed...